src/ripple/app/paths/cursor/ForwardLiquidityForAccount.cpp

```diff
@@ -46,6 +46,9 @@ TER PathCursor::forwardLiquidityForAccount () const
     auto const lastNodeIndex       = pathState_.nodes().size () - 1;
     auto viewJ = rippleCalc_.logs_.journal ("View");
 
+    STAmountCalcSwitchovers amountCalcSwitchovers (
+        rippleCalc_.view.info ().parentCloseTime);
```

Comment:
I will offer a comment for your consideration, since I'm hesitant to roil the waters this late in a review.  If you were to move the construction of `amountCalcSwitchovers` to just before it's first use, then a number of these conditional branches wouldn't need to construct it al all.  Just a possible optimization.